src/Form/Type/SimpleFormatterType.php

```diff
@@ -39,20 +38,20 @@ class SimpleFormatterType extends AbstractType
     private $templateManager;
 
     /**
-     * @param ConfigManagerInterface   $configManager   An Ivory CKEditor bundle configuration manager
-     * @param PluginManagerInterface   $pluginManager   An Ivory CKEditor bundle plugin manager
-     * @param TemplateManagerInterface $templateManager An Ivory CKEditor bundle template manager
+     * @param ConfigManagerInterface    $configManager   An Ivory CKEditor bundle configuration manager
+     * @param ?PluginManagerInterface   $pluginManager   An Ivory CKEditor bundle plugin manager
```

Comment:
This is wrong. Should be:

```
PluginManagerInterface|null   $pluginManager   An Ivory CKEditor bundle plugin manager
```

Reply:
Are you sure this is wrong? See https://github.com/phpDocumentor/TypeResolver/pull/35/files#diff-58912ee2ce6f273d72422833d8e0721fR48

Reply:
Never seen this before... What style is used inside symfony code?